ddtrace/appsec/_common_module_patches.py

```diff
@@ -27,8 +27,13 @@
 log = get_logger(__name__)
 _DD_ORIGINAL_ATTRIBUTES: Dict[Any, Any] = {}
 
+_is_patched = False
```

Comment:
We have the [`callonce`](https://github.com/DataDog/dd-trace-py/blob/13ec305f19f4e9bfb74dbd22c067e6704ee852db/ddtrace/internal/utils/cache.py#L114) decorator that can allow achieving the same thing perhaps

Reply:
but for test, we do some unpatching, so we would need a compatible mechanism for unpatch function